tf-modules/archive/clean_executions.tf

```diff
@@ -0,0 +1,53 @@
+resource "aws_sqs_queue" "clean_executions_dead_letter_queue" {
+  name                       = "${var.prefix}-cleanExecutionsDeadLetterQueue"
+  receive_wait_time_seconds  = 20
+  visibility_timeout_seconds = 60
+}
+
+resource "aws_lambda_function" "clean_executions" {
+  function_name    = "${var.prefix}-cleanExecutions"
+  filename         = "${path.module}/../../packages/api/dist/cleanExecutions/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../packages/api/dist/cleanExecutions/lambda.zip")
+  handler          = "index.handler"
+  role             = aws_iam_role.lambda_processing.arn
+  runtime          = "nodejs8.10"
+  timeout          = 900
+  memory_size      = 192
+  dead_letter_config {
+    target_arn = aws_sqs_queue.clean_executions_dead_letter_queue.arn
+  }
+  environment {
+    variables = {
+      CMR_ENVIRONMENT                           = var.cmr_environment
+      ExecutionsTable                           = var.dynamo_tables.Executions
+      stackName                                 = var.prefix
+      nonCompleteExecutionPayloadTimeout        = 30
```

Comment:
I'm pretty sure this was intended to be explicitly user configurable as part of the requirements.

Reply:
Just this value, or all four of the ExecutionPayload-related values?

Reply:
Fair, 'this' is nonspecific.   Yes, all four of them.

Reply:
I *don't* believe this has to be a deploy time configuration, however.   We could, instead of using env variables, have a persistent configuration store (e.g. a dynamo table) that gets written to and is user changeable. 

Upside: no more tying this sort of config to our deployment.   Downside: it makes *versioning* the config per lambda version super annoying.   Not sure that's actually a downside though :)